editor/test/integration/property_type_preservation_test.clj

```diff
@@ -207,28 +207,29 @@
 
 (defn- ensure-numeric-properties-preserve-type! [original-property-values]
   (with-clean-system
-    (let [original-meta {:version "original"}
-
-          property-values
-          (into (sorted-map)
-                (map (fn [[prop-kw prop-value]]
-                       (let [decorated-value (if (vector? prop-value)
-                                               (with-meta prop-value original-meta)
-                                               prop-value)]
-                         [prop-kw decorated-value])))
-                original-property-values)
-
-          graph-id (g/make-graph! :history true)
-          node-id (apply g/make-node! graph-id NumericPropertiesNode (mapcat identity property-values))]
-      (let [edit-type-by-prop-kw
+    (test-util/with-ui-run-later-rebound
```

Comment:
The only real change is wrapping the test in `with-ui-run-later-rebound`, everything else is indentation.

Reply:
I'm not sure why the original PR had no issues with this test...